pkg/grafana/client.go

```diff
@@ -102,7 +104,7 @@ func (c *Client) UpsertOrgUser(orgID int64, user gapi.User, role string) error {
 
 	err := c.AddOrgUser(orgID, user.Login, role)
 	if err != nil {
-		if !strings.Contains(err.Error(), "User is already member of this organization") {
+		if !strings.Contains(strings.ToLower(err.Error()), "user is already member of this organization") {
```

Comment:
The ErrOrgUserAlreadyAdded typed error exists in [8.2](https://github.com/grafana/grafana/blob/d7f71e9eaec57995d4537f758e300b31650d23ee/pkg/models/org_user.go#L14) and [6.5](https://github.com/grafana/grafana/blob/75f04c731f170918cfb4031ac059a7c7463f5b2e/pkg/models/org_user.go#L15) to use for string matching.

Reply:
This is what I see on those links:
8.2 ErrOrgUserAlreadyAdded = errors.New("user is already added to organization")
6.5 ErrOrgUserAlreadyAdded = errors.New("User is already added to organization")

but this kind of string whack-a-mole is largely what I want to avoid.

Reply:
Correct.  And the string we are matching is "is already member of this organization".  So I want to know where "already a member" came from, and if we are asserting this error correctly.